src/wiki/templates/wiki/edit.html

```diff
@@ -17,11 +17,34 @@
 
   <div class="col-lg-8">
     <form method="POST" class="form-horizontal" id="article_edit_form">
-      <input type="hidden" name="save" value="1" />
-      <input type="hidden" name="preview" value="1" />
       {% with edit_form as form %}
         {% include "wiki/includes/editor.html" %}
       {% endwith %}
+
+      <div class="form-group form-actions">
+        <div class="col-lg-8">
+          <div class="col-lg-offset-2">
+            <button class="btn btn-default" type="submit" name="preview" value="1" id="id_preview"
+                    formaction="{% url 'wiki:preview' path=urlpath.path article_id=article.id %}"
+                    formtarget="previewWindow"
+                    >
+              <span class="fa fa-eye"></span>
+              {% trans "Preview" %}
+            </button>
+            <button class="btn btn-primary" type="submit" name="save" value="1" id="id_save">
+              <span class="fa fa-check"></span>
+              {% trans "Save changes" %}
+            </button>
+            {% if article|can_delete:user %}
+              <a href="{% url 'wiki:delete' path=urlpath.path article_id=article.id %}" class="btn btn-danger pull-right">
+                <span class="fa fa-trash-o"></span>
+                {% trans "Delete article" %}
+              </a>
+            {% endif %}
+          </div>
+        </div>
+      </div>
+
```

Comment:
I remember now problems with `button` in IE6/IE7 - but according to [this](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button) they have been fixed since IE8, so only a fraction of a percent of users are affected now.